src/lib/components/error-pages/ErrorPage401.component.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import React from 'react';
+import {
+  ErrorPageContainer,
+  Row,
+  WarningIcon,
+  Title,
+  Description,
+  DescriptionContent,
+  Link,
+} from './ErrorPageStyle.jsx';
+import Button from '../buttonv2/Buttonv2.component';
+
+const translations = {
+  en: {
+    unexpected_error: 'Not authorized',
+    error_desc: `You don't have permission to view this page using the credentials that you supplied.`,
+    may_also_contact: 'You may contact ',
+    to_report_issue: ' to report this issue.',
+    return_home: 'Return Home',
+  },
+  fr: {
+    unexpected_error: `Vous n'êtes pas autorisé`,
+    error_desc: `Vous n'êtes pas autorisé à afficher cette page en utilisant les informations d'identification que vous avez fournies.`,
+    may_also_contact: 'Vous pouvez contacter le ',
+    to_report_issue: ' pour signaler ce problème.',
+    return_home: "Retour à l'accueil",
+  },
+};
+
+type Props = {
+  btnLink?: string,
+  supportLink?: string,
+  locale?: string,
+  history?: Object,
+};
+
+function ErrorPage401({
+  btnLink = '/',
+  supportLink = null,
+  locale = 'en',
+  history,
```

Comment:
i would rather have an `onReturnHome` function that let the app do whatever it want (history.push history.goBack window.reload) based on the context. rather than sending the history object and a link.
wdyt ?

Reply:
You are absolutely correct here! 
It's really cumbersome to pass the history object. 
I will just check we don't really use this `btnlink` for the moment, so with this change we won't introduce breaking change!